render/hyperscript.js

```diff
@@ -46,9 +61,7 @@ function execSelector(state, vnode) {
 				? state.attrs.className != null
 					? String(state.attrs.className) + " " + String(className)
 					: className
-				: state.attrs.className != null
-					? state.attrs.className
-					: null
+				: state.attrs.className
```

Comment:
Can you explain why this ternary is redundant? Specifically, how is it now incapable of returning `undefined`?